crates/viewer/re_chunk_store_ui/src/arrow_ui.rs

```diff
@@ -0,0 +1,54 @@
+use re_chunk_store::external::re_chunk::external::arrow2;
+use re_chunk_store::external::re_chunk::external::arrow2::array::Utf8Array;
+use re_viewer_context::UiLayout;
+
+//TODO(ab): this is copied/modified from `re_data_ui`. Consider unifying them?
```

Comment:
Also we really need to setup a registry for lossless text representations of arrow payloads based on semantic types.

E.g. I want `rerun.components.Color` to display as `0xFF88AAFF`, not the unintelligible integer it prints today.
And that's not specific to this view, it's everywhere we represent arrow as just lossless text (including terminal dumps).

Reply:
Agreed.

Alternatively, I would eventually like to have a toggle to apply the entire `re_data_ui` machinery for display. That could be useful eg. for interpreting enums (can those eventually be fully arrow?). That feature would have to be behind a cargo flag, such that `re_chunk_store_ui` can remain independent from the rerun viewer, should we want to make it its own binary.